JASP-Engine/JASP/R/common.R

```diff
@@ -1103,7 +1103,7 @@ saveImage <- function(plotName, format, height, width){
 
 }
 
-.newProgressbar <- function(ticks, callback, skim=5, response=FALSE) {
+.newProgressbar <- function(ticks, callback, skim=5, response=FALSE, parallel=FALSE) {
```

Comment:
Can you please change the function name?

Reply:
To what?

I chose this name to be inline with our footnotes:
footnotes <- .newFootnotes()
progressbar <- .newProgressbar()

It should be very clear that the function is a constructor. It returns an actual progressbar() function.